### PR TITLE
feat(electron): add electron-app

### PR DESCRIPTION
**Pull request by arokor**
Add basic electron skeleton

image

Generated with https://electron-vite.org/guide/

I had to lock down the electron version to an exact patch version in package.json. The reason for this is that there is logic for determining electron version in a postinstall script that fails when it is run in a yarn workspace.

### CODE REVIEW

**Comment by manast** [comment]
Nice. I guess it would also be quite useful to expose the version numbers of Electron, Chromium, etc as metadata for the player.

File: packages/platforms/electron-player/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@castmill/electron-player",
+  "version": "1.0.0",
+  "description": "Castmill Player Electron App",
+  "main": "./out/main/index.js",
+  "author": "example.com",
```
Comment (manast) [inline packages/platforms/electron-player/package.json:6]: "author": "Castmill AB"

**Review by manast — APPROVED** [review]
Just a few comments...

File: packages/platforms/electron-player/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+root = true
+
```
[the remark this answers is not in the document]
Comment (arokor) [reply packages/platforms/electron-player/.editorconfig:2]: Hmm. I guess this was generated by electron-vite. I'll remove it for now. We can always add it later if we feel a need for this

File: packages/platforms/electron-player/package.json (hunk 1 above)
Comment (arokor) [reply packages/platforms/electron-player/package.json:6]: yes!

**Pull request merged**